jstz_node/src/main.rs

```diff
@@ -46,14 +51,28 @@ async fn main() -> io::Result<()> {
 
     let rollup_client = Data::new(RollupClient::new(rollup_endpoint));
 
+    let (broadcaster, handler, cancel_read_file) =
+        LogService::init(args.kernel_file_path);
+
     HttpServer::new(move || {
         App::new()
             .app_data(rollup_client.clone())
+            .service(
+                Scope::new("/logs")
+                    .app_data(Data::from(broadcaster.clone()))
```

Comment:
Data needs to be created outside the closure of `HttoServer::new` -- see https://docs.rs/actix-web/latest/actix_web/struct.App.html#shared-mutable-state

Reply:
Additionally, can this be added to the top-level `.app_data` (for consistency)?

Reply:
> Data needs to be created outside the closure of `HttoServer::new` -- see https://docs.rs/actix-web/latest/actix_web/struct.App.html#shared-mutable-state

The broadcaster is created outside of Server, it is of `Arc` so can be shared across workers :)

Reply:
> Additionally, can this be added to the top-level `.app_data` (for consistency)?

hmm in that case other services like operation and account can access the broadcaster. I thought it would be better to limit the scope? wdyt?